src/linter/html/transpiler.ts

```diff
@@ -52,6 +54,23 @@ export default async function transpileHtml(
 	}
 }
 
+function detectTestStarter(resourcePath: ResourcePath, scriptTags: Tag[], context: LinterContext) {
+	const shouldBeMigrated = scriptTags.some((tag) => {
+		return (resourcePath.includes("testsuite.qunit.html") && !tag.attributes.some((attr) => {
+			return attr.name.value.toLowerCase() === "src" &&
+				attr.value.value.includes("resources/sap/ui/test/starter/createSuite.js");
```

Comment:
`test-resources/sap/ui/test/starter/createSuite.js` would also match here, but would be wrong. To be correct, it should be checked to be equal to `resources/sap/ui/test/starter/createSuite.js` or ending with `/resources/sap/ui/test/starter/createSuite.js` (like in `../../../resources/sap/ui/test/starter/createSuite.js`), right?

Reply:
Thanks!